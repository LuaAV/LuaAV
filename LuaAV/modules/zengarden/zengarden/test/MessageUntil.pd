#N canvas 0 22 450 300 10;
#X obj 52 34 loadbang;
#X obj 121 35 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 52 74 t b b b;
#X msg 141 138 3;
#X msg 172 139 8;
#X obj 183 207 print eight;
#X obj 141 234 print three;
#X obj 183 180 until;
#X obj 141 195 until;
#X obj 25 153 until;
#X obj 25 175 f;
#X obj 54 175 + 1;
#X obj 100 177 sel 0;
#X obj 25 219 print;
#X obj 54 197 mod 10;
#X connect 0 0 2 0;
#X connect 1 0 2 0;
#X connect 2 0 9 0;
#X connect 2 1 3 0;
#X connect 2 2 4 0;
#X connect 3 0 8 0;
#X connect 4 0 7 0;
#X connect 7 0 5 0;
#X connect 8 0 6 0;
#X connect 9 0 10 0;
#X connect 10 0 11 0;
#X connect 10 0 13 0;
#X connect 11 0 14 0;
#X connect 12 0 9 1;
#X connect 14 0 10 1;
#X connect 14 0 12 0;
