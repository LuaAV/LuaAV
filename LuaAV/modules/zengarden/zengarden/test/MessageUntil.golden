[@ 0.000ms] eight: bang
[@ 0.000ms] eight: bang
[@ 0.000ms] eight: bang
[@ 0.000ms] eight: bang
[@ 0.000ms] eight: bang
[@ 0.000ms] eight: bang
[@ 0.000ms] eight: bang
[@ 0.000ms] eight: bang
[@ 0.000ms] three: bang
[@ 0.000ms] three: bang
[@ 0.000ms] three: bang
[@ 0.000ms] print: 0
[@ 0.000ms] print: 1
[@ 0.000ms] print: 2
[@ 0.000ms] print: 3
[@ 0.000ms] print: 4
[@ 0.000ms] print: 5
[@ 0.000ms] print: 6
[@ 0.000ms] print: 7
[@ 0.000ms] print: 8
[@ 0.000ms] print: 9